com.unity.shadergraph/Editor/Drawing/MaterialGraphEditWindow.cs

```diff
@@ -887,6 +888,19 @@ public void Initialize(string assetGuid)
             }
         }
 
+        private bool PromptSaveIfDirtyOnQuit()
```

Comment:
Can we register our existing OnDestroy for this call instead of duplicating the function body? Or make a third function that this and OnDestroy both call?

Reply:
For sure, easy enough